x/supply/alias.go

```diff
@@ -18,6 +18,7 @@ const (
 	Basic        = types.Basic
 	Minter       = types.Minter
 	Burner       = types.Burner
+	Staking      = types.Staking
```

Comment:
Why was this new permission type added? I don't see this permission added in `simapp.go`? Also, we should update the spec doc too.

/cc @fedekunze

Reply:
https://github.com/cosmos/cosmos-sdk/pull/4255/files#r297838913

Reply:
@alexanderbez to allow for permission checks when a module account tries to delegate or undelegate 

https://github.com/cosmos/cosmos-sdk/pull/4255#discussion_r297838913